code/src/containers/forms/product-form/components/main-info/MainInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+
+import { productLocales } from "@/containers/forms/product-form/ProductForm.constants";
+import { ProductFormMainInfoSectionProps } from "@/containers/forms/product-form/ProductForm.types";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppButton from "@/components/app-button/AppButton";
+import AppInput from "@/components/app-input/AppInput";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import cn from "@/utils/cn/cn";
+
+const MainInfo = ({ register, errors }: ProductFormMainInfoSectionProps) => {
+  const [selectedLocal, setSelectedLocal] = useState(0);
```

Comment:
Provide better name for this state, it is really confusing

Reply:
Thank you, I changed it to `selectedLocaleIndex`